### ISSUE TITLE
If only one destination left, halt and report

### ISSUE THREAD

**Issue by gangadevib** [CS4244/Where2Go#4]

**Comment by Dauth** [comment]
Currently this is what has been done
1) count all destination facts on start
2) for any filter, the totalDestination global must be decremented
3) Once the totalDestination is < 2, it will delete all ASK  fact. 

So we can churn out a report template to tell if pass/fail?
0 destination = fail
1 destination = pass
2+ destination = need to add more questions?

**Comment by gangadevib** [comment]
We can use a report template to report to the user. 
0 destination = fail -> meaning we inform the user that there are no destination and he will have to restart the entire process again?
2+destination -> If there are more questions left then continue, else if report any one of the destinations?

**Comment by Dauth** [comment]
Yes please use the global var for destination for the LHS checking in the report, it keeps tracks of the destination's count as long as all filters decrement it after retract the destination fact
